docs/explanation/gno-tooling/cli/gnofaucet.md

```diff
@@ -0,0 +1,53 @@
+---
+id: gno-tooling-gnofaucet
+---
+
+# gnofaucet
+
+`gnofaucet` is a server for distributing GNOT, the gas currency of Gnoland, to specific addresses in a local chain. Interact with the `gnofaucet` from an address with an empty balance in your locally built testnet to fuel it with GNOT to pay for transactions.
+
+## Run `gnofaucet` Commands
+
+Enable the faucet using the following command.
+
+```bash
+gnofaucet serve
+```
+
+#### **Options**
+
+| Name                      | Type    | Description                                                                          |
+| ------------------------- | ------- | ------------------------------------------------------------------------------------ |
+| `chain-id`                | String  | The id of the chain (required).                                                      |
+| `gas-wanted`              | Int64   | The maximum amount of gas to use for the transaction (default: `50000`)              |
+| `gas-fee`                 | String  | The gas fee to pay for the transaction.                                              |
+| `memo`                    | String  | Any descriptive text (default: `""`)                                                 |
+| `test-to`                 | String  | Test address (optional) 부연설명 필요                                                |
+| `send`                    | String  | Coins to send (default: `"1000000ugnot"`).                                           |
+| `captcha-secret`          | String  | The secret key for the recaptcha. If empty, the captcha is disabled (default: `""`). |
+| `is-behind-proxy`         | Boolean | Uses X-Forwarded-For IP for throttling (default: `false`).                           |
+| `insecure-password-stdin` | Boolean | INSECURE! Takes password from stdin (default: `false`).                              |
+
+## Example
+
+### **Step 1. Create an account named `test1` with the test seed phrase below.**
+
+```bash
+gnokey add test1 --recover
+```
+
+> **Test Seed Phrase:** source bonus chronic canvas draft south burst lottery vacant surface solve popular case indicate oppose farm nothing bullet exhibit title speed wink action roast
+
+### **Step 2. Run `gnofaucet`**
+
+```bash
+gnofaucet serve test1 --chain-id dev --send 500000000ugnot
+```
+
+### **Step 3. Receive GNOTs from the faucet**
+
+```bash
```

Comment:
If there is a form, I suggest prioritizing opening the browser as the first option before suggesting the curl method.

Reply:
Added:

[969e914](https://github.com/gnolang/gno/pull/1046/commits/969e9147bcce28ac49f777f9f1d10a21fa28aab0)